packages/legacy/core/App/localization/fr/index.ts

```diff
@@ -97,17 +97,22 @@ const translation = {
         "Message1032": "Un problème est survenu lors du retrait du justificatif.",
         "Title1033": "Impossible de récupérer le justificatif.",
         "Message1033": "Le justificatif est introuvable.",
-        "Title1034": "Unable to fetch proof request. (FR)",
-        "Message1034": "Proof request could not be found. (FR)",
-        "Title1035": "Unable to fetch credential offer. (FR)",
-        "Message1035": "Credential offer could not be found. (FR)",
-        "Title1036": "Unable to fetch wallet credentials (FR)",
-        "Message1036": "Wallet credentials could not be found (FR)",
-        "Title1037": "Unable to remove contact. (FR)",
-        "Message1037": "There was a problem while removing the contact. (FR)",
+        "Title1034": "Impossible d'obtenir une demande de preuve.",
+        "Message1034": "La demande de preuve est introuvable.",
+        "Title1035": "Impossible de récupérer l'offre d'attestation.",
+        "Message1035": "L'offre d'attestation est introuvable.",
+        "Title1036": "Impossible de récupérer les justificatifs d'identité du portefeuille",
+        "Message1036": "Les justificatifs d'identité du portefeuille sont introuvables",
```

Comment:
On utilise attestation (1035) et directement après on utilise justificatifs (1036)... Il manque peut-être de consistance ?

We use "attestation" (1035) and directly after we use "justificatifs" (1036)... Perhaps it lacks consistency?